portfolio/src/main/webapp/index.html

```diff
@@ -45,7 +45,33 @@
                 <img class="center" src="images/chicago-sleep.jpg"/>
                 <p>And sleeping in odd positions.</p>
             </div>
+            <div id="bottom">
+                <p>Please feel free to drop some questions, comments, and concerns if you have any:</p>
+                <form action="/my-CQCs" method="POST">
+                    <input type="text" name="email-input" placeholder="Your Email"/>
+                    <br/><br/>
```

Comment:
Can you use `<br>` as opposed to `<br/>`?

Context:
https://google.github.io/styleguide/htmlcssguide.html#Document_Type